core/domain/exp_domain.py

```diff
@@ -111,6 +111,12 @@
 CMD_MIGRATE_STATES_SCHEMA_TO_LATEST_VERSION = (
     'migrate_states_schema_to_latest_version')
 
+# Possible allowed commands for exploration change.
+LIST_CMD_EXPLORATION_CHANGE = [
+    CMD_ADD_WRITTEN_TRANSLATION,
+    CMD_EDIT_STATE_PROPERTY
+]
```

Comment:
I don't understand why we need this list? There are more allowed commands for the exploration change.

Reply:
The [allowed commands](https://github.com/chiragbaid7/oppia/blob/ca8ded1a3e1fb29bca38130cdc94a49cc00258ef/core/domain/exp_domain.py#L281) list is initialized inside the class `ExplorationChange` same for other domains. So in order to check for any allowed command for a domain I suggest the following approach with this the initializing of a list at the global scope will not be required.

```
    if obj.get('cmd') is None:
        raise base.BaseHandler.InvalidInputException(
            'Missing cmd key in change dict')
    else:
        ExplorationChange_allowed_cmds = [command['name'] for command in 
            exp_domain.ExplorationChange.ALLOWED_COMMANDS]
        QuestionChange_allowed_cmds = [command['name'] for command in 
            question_domain.QuestionChange.ALLOWED_COMMANDS]

        if obj['cmd'] in ExplorationChange_allowed_cmds:
            exp_domain.ExplorationChange(obj)
        elif obj['cmd'] in QuestionChange_allowed_cmds:
            question_domain.QuestionSuggestionChange(obj)
        else:
            raise base.BaseHandler.InvalidInputException(
                '%s cmd is not allowed.' % obj['cmd'])
```